<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWoPlaybookWidget</class>
 <widget class="QWidget" name="QWoPlaybookWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="debugArea" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnBrowserScript">
          <property name="text">
           <string>Open script directory</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../private/skins/black/edit.png</normaloff>../private/skins/black/edit.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(224, 27, 36);</string>
          </property>
          <property name="text">
           <string>Suggest using Qt tool to edit related script files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radAuto">
          <property name="text">
           <string>Auto reflesh</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radManual">
          <property name="text">
           <string>Manual reflesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnReload">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../private/skins/black/reload.png</normaloff>../private/skins/black/reload.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnClose">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../private/skins/black/close.png</normaloff>../private/skins/black/close.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
